skyportal/handlers/api/internal/source_counts.py

```diff
@@ -0,0 +1,32 @@
+import datetime
+from sqlalchemy import func
+from baselayer.app.access import auth_or_token
+from ...base import BaseHandler
+from ....models import DBSession, Source
+
+default_prefs = {'sinceDaysAgo': 7}
```

Comment:
Necessary to have this as a preference? Can we just hard code to a week for simplicity? Doesn't seem like this is a very important scientific factoid.

Reply:
I think it's fine to leave it in. Some people may want to see just in the last day, others the last week.